packages/contracts-ops/contracts/scripts/UpgradeCallBatches.s.sol

```diff
@@ -21,32 +21,23 @@ contract UpgradeCallBatchLogic is Script, Config, CallBatch {
 
     function pushUpgrade(string memory _domain) internal {
         currentDomain = _domain;
-        console2.log("upgrade ", _domain);
+        pushSingleUpgrade(governanceRouterUpgrade(currentDomain));
+        pushSingleUpgrade(bridgeRouterUpgrade(currentDomain));
+        pushSingleUpgrade(bridgeTokenUpgrade(currentDomain));
+        pushSingleUpgrade(tokenRegistryUpgrade(currentDomain));
+        pushSingleUpgrade(homeUpgrade(currentDomain));
         pushSingleUpgrade(
-            governanceRouterUpgrade(currentDomain),
-            "governanceRouter"
-        );
-        pushSingleUpgrade(bridgeRouterUpgrade(currentDomain), "bridgeRouter");
-        pushSingleUpgrade(bridgeTokenUpgrade(currentDomain), "bridgeToken");
-        pushSingleUpgrade(tokenRegistryUpgrade(currentDomain), "tokenRegistry");
-        pushSingleUpgrade(homeUpgrade(currentDomain), "home");
-        pushSingleUpgrade(
-            replicaOfUpgrade(currentDomain, getConnections(currentDomain)[0]),
-            "replica"
+            replicaOfUpgrade(currentDomain, getConnections(currentDomain)[0])
         );
     }
 
-    function pushSingleUpgrade(
-        Upgrade memory _upgrade,
-        string memory contractName
-    ) private {
+    function pushSingleUpgrade(Upgrade memory _upgrade) internal {
         // check if upgrade is unnecessary
         (, bytes memory result) = address(_upgrade.beacon).call("");
```

Comment:
Should this be a call (wrt broadcast)? Perhaps easier to do the vm.load() I mentioned elsewhere